### PR TITLE
Commit exam1

### PR DESCRIPTION
**Pull request by vuchungdung**
review code exam1

### CODE REVIEW

File: VuChungDung/style.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+.main-detail__link{
+    color: black;
+    font-weight: 600;
+}
+.main-detail__footer{
+    display: flex;
+    align-items: center;
+}
+.main-detail__footer--text{
+    color: rgba(255, 102, 0, 0.884);
```
Comment (dinhhung09138) [inline VuChungDung/style.css]: I prefer use Hex color code to decimal color code.
Fix for whole page.

File: VuChungDung/style.css [hunk 2]
```diff
@@ -0,0 +1,122 @@
+*{
+    margin:  0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+    text-decoration: none;
+}
```
Comment (dinhhung09138) [inline VuChungDung/style.css:7]: Should add blank line to separate between group

File: VuChungDung/style.css (hunk 1 above)
Comment (vuchungdung) [reply VuChungDung/style.css]: thanks for reviewing

File: VuChungDung/index.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                            <h2 class="main-detail__footer--text">DO IT NOW</h2>
+                            <button type="submit" class="main-detail__footer--btn">click here!</button>
+                        </div>
+                    </div> 
+                </div>
+            </section>
+            <section class="main-items">
+                <div class="main-item__detail left">
+                    <div class="main-item__img">
+                        <img src="./img/img-small.jpg" alt="" class="main-item__img--left">
```
Comment (dinhhung09138) [inline VuChungDung/index.html]: Don't use double dash (--) in CSS

**Comment by dinhhung09138** [comment]
Done

**Review by dinhhung09138 — APPROVED** [review]
Done

**Pull request merged**